onadata/apps/api/tests/viewsets/test_submission_review_viewset.py

```diff
@@ -351,3 +351,23 @@ def test_submission_review_created_by_filter(self):
         self.assertEqual(200, response.status_code)
         self.assertEqual(1, len(response.data))
         self.assertEqual(review_two_data['id'], response.data[0]['id'])
+
+    def test_bulk_create_approved_review_missiong_note(self):
+        """
+        Test that we can bulk create approved submission reviews without a note
+        """
+        instances = self.xform.instances.all()
+        submission_data = [
+            {
+                'instance': _.id,
+                'status': SubmissionReview.APPROVED
+            } for _ in instances
+        ]
```

Comment:
If I recall correctly, a user would select a group of submissions and either approve them or reject all of them. The note made will apply to all selected submissions. Would this mean that we need a list of submission IDs and a single status that applies to all? Or are we expecting each submission ID will have a status that may not be the same as the rest?

Reply:
I think it means the selected submission IDs would all have the same single status. Unless I'm mistaken @moshthepitt

Reply:
I think it would be best to allow for bulk creation of the SubmissionReviews regardless of consistency in the status. All data in the list submitted to be processed individually. Hence allowing bulk creation with different statuses.

Having a list of submission IDs and a single status for all will be a little restrictive from our end I think.